core/mps/default_psr.go

```diff
@@ -39,6 +39,9 @@ func NewDefaultPrivateStateRepository(db ethdb.Database, cache state.Database, p
 }
 
 func (dpsr *DefaultPrivateStateRepository) DefaultState() (*state.StateDB, error) {
+	if dpsr == nil {
```

Comment:
I've tried to reproduce a situation where one is able to invoke `dpsr.DefaultState()` with `dpsr` being nil but couldn't figure out how (I get the panic before getting inside the `DefaultState()` method).

Reply:
I don't know, it seems impossible.
Yet it is reproducable, and I asked to have the same env in order to reproduce the error.
https://github.com/ConsenSys/quorum/issues/1287?id=issuecomment-1009834186#issuecomment-1007232132